tests/integration/pilot/testdata/generate-cert.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Same for the cert generation - I think we have a makefile entry to generate the other ones, we use them in the VM test. 

And we use the certgen tool in security/tools